CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
         ([#3048](https://github.com/Automattic/pocket-casts-android/pull/3048))
     *   Add support for the Catalan laguage
         ([#3072](https://github.com/Automattic/pocket-casts-android/pull/3072))
+*   Bug Fixes
+    *   Fix effects being lost when paused
+        ([#3084](https://github.com/Automattic/pocket-casts-android/pull/3084))
 *   Updates
```

Comment:
@geekygecko I've moved this PR to 7.77. Please, make sure you update the changelog of this PR to 7.77 too